### PR TITLE
Updated Adoption Page

### LINKED ISSUE #135: [Bug Report] - Merge in changes to adoptions into the current Uptane page
### Describe the bug.

There were newer changes to the adoptions page that didn't make it over into the new site.
https://github.com/uptane/uptane.github.io/blob/362e7e9b92031d18c1afe2985f83c57637770c36/adoptions.md?plain=1

Merge in changes from here.

### Describe the steps to reproduce the behavior.

_No response_

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating environment

_No response_

### What browsers are you seeing the problem on?

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by devilkiller-ag**
# Summary of Changes
Updated the Adoption Page

## Related Issue
Closes #135 

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have added or updated test cases to ensure the code's functionality.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

## Reviewer(s)
@hexsecs @Abhijay007

### CODE REVIEW

**Review by JustinCappos — DISMISSED** [review]
This looks fine to me, but would like @tkfu to review before merging if possible...

File: bun.lockb
Comment (hexsecs) [inline bun.lockb:1]: I don't think this file should be added.

**Review by hexsecs — CHANGES_REQUESTED** [review]
I think you need to remove the bun.lockb file. I don't think it's necessary.

File: bun.lockb
Comment (devilkiller-ag) [reply bun.lockb:1]: Sorry, Actually I was trying bun js instead of node js. Should I add this to gitignore as most probably many people will try to use bun js instead of node js in upcoming years.

Comment (hexsecs) [reply bun.lockb:1]: I just tried building with bun. It seems to work fine. Either way let's just support building with yarn and npm for now. It shouldn't be part of this pull request.

**Comment by devilkiller-ag** [comment]
I mistakenly dismissed @JustinCappos's review. Kindly, forgive me.

**Review by hexsecs — APPROVED** [review]

**Pull request merged**
